addons/product/models/product_template_attribute_value.py

```diff
@@ -110,6 +110,7 @@ def write(self, values):
                         product=ptav.product_tmpl_id.display_name,
                     ))
         res = super().write(values)
+        self.env.registry.clear_cache()
```

Comment:
do we need to do that in the product module ? I think the bug only affects the e commerce. It should be in website_sale module.